ci-operator/step-registry/telcov10n/system/telco-kpis/reporting/generate-report/telcov10n-system-telco-kpis-reporting-generate-report-commands.sh

```
@@ -36,14 +37,36 @@ main() {
+            SPLUNK_FLAG="${SPLUNK_FLAG} -e splunk_ci_build_number=${BUILD_ID:-unknown}"
+        fi
+    fi
+
+    echo "Running generate-report playbook (development_mode: ${DEVELOPMENT_MODE}, splunk: ${PUSH_ENABLED:-false})"
     ansible-playbook ./playbooks/telco-kpis/generate-report.yml \
         -i ./inventories/ocp-deployment/build-inventory.py \
         -e spoke_cluster="${SPOKE_CLUSTER}" \
         -e output_filename="${OUTPUT_FILENAME}" \
         -e timestamp="${TIMESTAMP}" \
         -e development_mode="${DEVELOPMENT_MODE}" \
         ${FILTER_FLAG} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)